### PR TITLE
fix: route list search query string

### LINKED ISSUE #1186: bug: route's label search is return 400
# Bug report

## Describe the bug
the lable of route is `a:1`
when search on route, is return 400.

```
Request URL: http://127.0.0.1:9000/apisix/admin/routes?label=a%3A1%2C&page=1&page_size=20
Request Method: GET
Status Code: 400 Bad Request
Response: {"code":10000,"message":"malformed label: \"a:1,\"","data":null,"request_id":"23661bdf-dbd4-47f1-9a7c-d0c5783a4cbe"}
```
## How to Reproduce
image

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
resolve #1186 
___
### Bugfix
- Description

### CODE REVIEW

**Review by moonming — CHANGES_REQUESTED** [review]
need test cases to cover

File: web/cypress/fixtures/selector.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-    "notification": ".ant-notification-notice-message"
+  "rcVirtualList": ".rc-virtual-list",
+  "notification": ".ant-notification-notice-message",
+  "antDrawerWrapperBody": ".ant-drawer-wrapper-body"
```
Comment (juzhiyuan) [inline web/cypress/fixtures/selector.json]:
```suggestion
  "drawerBody": ".ant-drawer-wrapper-body"
```

File: web/cypress/fixtures/selector.json [hunk 2]
```diff
@@ -1,3 +1,5 @@
 {
-    "notification": ".ant-notification-notice-message"
+  "rcVirtualList": ".rc-virtual-list",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/selector.json]:
```suggestion
  "selector": ".rc-virtual-list",
```

**Review by juzhiyuan — APPROVED** [review]

**Comment by LiteSun** [comment]
> need test cases to cover

The test case for search label has been added.

**Review by moonming — APPROVED** [review]

**Pull request merged**